src/api-v1.js

```diff
@@ -580,6 +580,197 @@ api.declare({
 
 api.declare({
   method: 'put',
+  route: '/ami-set/:id',
+  name: 'createAmiSet',
+  input: 'create-ami-set-request.json#',
+  deferAuth: true,
+  scopes: [['aws-provisioner:manage-ami-set:<amiSetId>']],
+  title: 'Create new AMI Set',
+  stability:  base.API.stability.stable,
+  description: [
+    'Create an AMI Set. An AMI Set is a collection of AMIs with a single name.',
+  ].join('\n'),
+}, async function (req, res) {
+  let input = req.body;
+  let id = req.params.id;
+
+  input.lastModified = new Date();
```

Comment:
We shouldn't really be setting properties on the request body object.  Also, this value isn't actually used in the `AmiSet.create` method call below